tox.ini

```diff
@@ -7,7 +7,7 @@ deps =
    git+https://github.com/pycqa/astroid@master
    isort
 
-commands = pylint -rn --rcfile={toxinidir}/pylintrc --load-plugins=pylint.extensions.docparams, pylint.extensions.mccabe {envsitepackagesdir}/pylint
+commands = pylint -rn --rcfile={toxinidir}/pylintrc --load-plugins=pylint.extensions.docparams,pylint.extensions.mccabe {envsitepackagesdir}/pylint
```

Comment:
if lack of previous warnings was caused by this bug in tox.ini, please revert it for now. We can tackle this problem in another commit.

Reply:
As you want